cg/constants/tags.py

```diff
@@ -1,145 +1,10 @@
 """Tags for storing analyses in Housekeeper"""
 
 HK_TAGS = {
-    "wes": ["mip-dna", "wes"],
-    "wgs": ["mip-dna", "wgs"],
-    "wts": ["mip-rna"],
     "microsalt": ["microsalt"],
 }
 HK_FASTQ_TAGS = ["fastq"]
 
-# used to convert MIP tags derived from the deliverables to MIP standard tags and to check for
```

Comment:
I dont think any of the microsalt tags are in use. Maybe worth checking if the entire tag file could be deleted??

Reply:
I was tempted, but did not dare. However, if you also think it is a good idea I will do it

Reply:
My delete button is superfast (sometimes in the past... a bit too fast 😈 )

Reply:
done